### ISSUE TITLE
Path Check BT crashes when executed

### ISSUE THREAD

**Issue by crayne** [tripleblindmarket/private-kit#1221]
## Issue type

Environment

## Version 
- Release build version: 
1.22.4
## Affected Devices

| Brand        |  Model        | OS  |
| ------------ |:-------------:| ----:|

Samsung Galaxy S8 emulator
Samsung Galaxy S9 device

## Steps to reproduce
yarn run-android-bt

## Expected Behavior
Normal onboarding

- When I click `Stop Logging` I am taken to the `Import` screen. Expect to
  go back to the Welcome greeting.

![image](https://user-images.githubusercontent.com/134657/87263579-9656f180-c48b-11ea-9a8b-bff828c5a3f6.png)

**Comment by johnschoeman** [comment]
Looks like an issue with `react-native-config`, can you confirm you have a local `.env.bt` file with `TRACING_STRATEGY=bt` ?

**Comment by crayne** [comment]
Yes, I can.  A quick fix would be to set Config.TRACING_STRATEGY to bt if it is undefined.  Would that be unacceptably ugly?

**Comment by johnschoeman** [comment]
> Yes, I can. A quick fix would be to set Config.TRACING_STRATEGY to bt if it is undefined. Would that be unacceptably ugly?

This is probably not the way we want to go here.  Do you have a local `.env.bt` file with `TRACING_STRATEGY=bt` ?  and can you confirm if `react-native-config` is working properly for you?  something like 
```
import Config from 'react-native-config'
console.log(Config)
```
should be sufficient.

**Comment by crayne** [comment]
I do have a local .env.bt file with TRACING_STRATEGY=bt.  The result of

````
import Config from 'react-native-config'
console.log("Config.TRACING_STRATEGY is " + Config.TRACING_STRATEGY);
````
is
 LOG  Config.TRACING_STRATEGY is undefined
